app-emulation/bochs/bochs-2.6.11.recipe

```diff
@@ -1,12 +1,13 @@
 SUMMARY="The open-source IA-32 emulator"
-DESCRIPTION="Bochs IA-32 Emulator provides a virtual PC that can run operating systems such as Windows, Linux, and BSD"
+DESCRIPTION="Bochs IA-32 Emulator provides a virtual PC that can run \
+operating systems such as Windows, Linux, and BSD"
 HOMEPAGE="http://bochs.sourceforge.net/"
-COPYRIGHT="2001-2017 The Bochs Project"
+COPYRIGHT="2001-2020 The Bochs Project"
 LICENSE="GNU LGPL v2.1
 	Bochs"
 REVISION="4"
```

Comment:
REVISION should be "1" here as it's a newer version

Reply:
Whoops, I missed that.